wls-basisdaten-service/src/test/java/de/muenchen/oss/wahllokalsystem/basisdatenservice/services/wahltag/WahltageServiceTest.java

```diff
@@ -0,0 +1,67 @@
+package de.muenchen.oss.wahllokalsystem.basisdatenservice.services.wahltag;
+
+import org.assertj.core.api.Assertions;
+import de.muenchen.oss.wahllokalsystem.basisdatenservice.domain.Wahltag;
+import de.muenchen.oss.wahllokalsystem.basisdatenservice.domain.WahltagRepository;
+import java.time.LocalDate;
+import java.util.List;
+import lombok.val;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+class WahltageServiceTest {
+
+    @Mock
+    WahltagRepository wahltagRepository;
+    @Mock
+    WahltagModelMapper wahltagModelMapper;
+    @Mock
+    WahltageClient wahltageClient;
+
+    @InjectMocks
+    WahltageService unitUnderTest;
+
+    @Nested
+    class GetWahltage {
+
+        @Test
+        void dataIsLoadedFromRemoteEvenIfExistingInRepoAndRepoIsUpdated() {
+            val mockedListOfEntities = createWahltagList("2");
+            val mockedClientResponse = createWahltagModelList("2");
+            val mockedMappedSavedEntities = createWahltagModelList("2");
+
+            Mockito.when(wahltageClient.getWahltage(LocalDate.now().minusMonths(3))).thenReturn(mockedClientResponse);
+            Mockito.when(wahltagModelMapper.fromWahltagModelToWahltagEntityList(mockedClientResponse)).thenReturn(mockedListOfEntities);
+            Mockito.when(wahltagRepository.saveAll(mockedListOfEntities)).thenReturn(mockedListOfEntities);
+            Mockito.when(wahltagRepository.findAllByOrderByWahltagAsc()).thenReturn(mockedListOfEntities);
+            Mockito.when(wahltagModelMapper.fromWahltagEntityToWahltagModelList(mockedListOfEntities)).thenReturn(mockedMappedSavedEntities);
+
+            val result = unitUnderTest.getWahltage();
+            Assertions.assertThat(result).isSameAs(mockedMappedSavedEntities);
+            Mockito.verify(wahltagRepository).saveAll(mockedListOfEntities);
+
+        }
+
+        private List<Wahltag> createWahltagList(String pIndex) {
```

Comment:
warum der Prefix `p`? `wahltagIDPrefix` oder ähnliches wäre aus meiner Sicht auch eine bessere Bezeichnung. Gleiches gilt für `createWahltagModelList`

Reply:
p vom Parameter

Reply:
da macht man meines Wissens so nicht mehr. Ein Name für Variable sollte eher selbstklärend sein als den Typ oder die Art preiszugeben. Ob etwas eine lokale Variable, Parameter oder Objektvariable ist spielt meist keine Rolle. Klassenvariablen sind in der Regelkonstanten und die sind an UPPER_CASE erkennbar. Durch IDEs wird schnell erkennbar wo die Quelle einer Variablen liegt.

Daher bin ich dafür das der Prefix `p` für Parameter verschwindet.